client/src/components/timeline/ZoomControls.vue

```diff
@@ -3,6 +3,9 @@
     <v-btn fab :disabled="!canZoomOut" @click="$emit('zoom', -1)">
       <v-icon>mdi-magnify-minus-outline</v-icon>
     </v-btn>
+    <v-btn @click="$emit('jump')">
+      <v-icon>mdi-magnify</v-icon>   
```

Comment:
Isn’t there a more appropriate icon than a magnifying glass?

Reply:
Do you have any other ideas ?